type/__packages/man.rst

```diff
@@ -0,0 +1,73 @@
+cdist-type__packages(7)
+=======================
+
+NAME
+----
+cdist-type__packages - Install or remove packages.
+
+
+DESCRIPTION
+-----------
+This type calls `__package` only if a package needs to be installed or
```

Comment:
Nitpick: `` `__package` `` is Markdown. I think it should be ``:strong:`cdist-type__package`\ (7)`` to be consistent with other `man.rst`.